tests/src/test/java/com/maxio/advancedbilling/controllers/coupons/CouponsControllerListTest.java

```diff
@@ -0,0 +1,180 @@
+package com.maxio.advancedbilling.controllers.coupons;
+
+import com.maxio.advancedbilling.TestClient;
+import com.maxio.advancedbilling.exceptions.ApiException;
+import com.maxio.advancedbilling.models.BasicDateField;
+import com.maxio.advancedbilling.models.Coupon;
+import com.maxio.advancedbilling.models.CouponResponse;
+import com.maxio.advancedbilling.models.ListCouponsFilter;
+import com.maxio.advancedbilling.models.ListCouponsInput;
+import com.maxio.advancedbilling.models.ProductFamily;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.maxio.advancedbilling.utils.assertions.CommonAssertions.assertUnauthorized;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CouponsControllerListTest extends CouponsControllerTestBase {
+
+    protected static ProductFamily productFamily2;
+    protected static ProductFamily productFamily3;
+    protected static List<Coupon> coupons = new ArrayList<>();
+    protected static ZonedDateTime testStart;
+
+    @BeforeAll
+    static void setupResources() throws IOException, ApiException, InterruptedException {
+        Thread.sleep(2000);
+        testStart = ZonedDateTime.now();
+        Thread.sleep(2000);
+
+        productFamily2 = TEST_SETUP.createProductFamily();
+        productFamily3 = TEST_SETUP.createProductFamily();
+        for (int i=0; i<4; i++) {
+            coupons.add(COUPONS_CONTROLLER.createCoupon(productFamilyId, validCouponRequest()).getCoupon());
+        }
+        for (int i=0; i<3; i++) {
+            coupons.add(COUPONS_CONTROLLER.createCoupon(productFamily2.getId(), validCouponRequest()).getCoupon());
+        }
+        for (int i=0; i<3; i++) {
+            coupons.add(COUPONS_CONTROLLER.createCoupon(productFamily3.getId(), validCouponRequest()).getCoupon());
+        }
+    }
+
+    @Test
+    void shouldListCouponsWithoutFilters() throws IOException, ApiException {
+        List<Coupon> listedCoupons  = COUPONS_CONTROLLER.listCoupons(new ListCouponsInput())
+                .stream().map(CouponResponse::getCoupon).toList();
+        // can't narrow down to specific coupons when listing without filters so just asserting response is not empty
```

Comment:
You can add assertion for size isAtLeast(10) instead of isNotEmpty, as you know the minimum size of coupons which should be present.